pg-testing/src/test/java/org/folio/tlib/postgres/testing/TenantPgPoolContainerTest.java

```diff
@@ -0,0 +1,42 @@
+package org.folio.tlib.postgres.testing;
+
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.Matchers.is;
+
+import java.util.Map;
+import java.util.stream.Stream;
+
+import org.folio.tlib.postgres.TenantPgPool;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.Arguments;
+import org.junit.jupiter.params.provider.MethodSource;
+
+class TenantPgPoolContainerTest {
+
+  /**
+   * Assert
+   * <a href="https://github.com/testcontainers/testcontainers-java/blob/main/modules/postgresql/src/main/java/org/testcontainers/containers/PostgreSQLContainer.java">
+   * PostgreSQLContainer.java</a> default.
+   */
+  @Test
+  void defaultConnectOptions() {
+    assertThat(TenantPgPool.getDefaultConnectOptions().getDatabase(), is("db"));
+    TenantPgPoolContainer.create();
+    assertThat(TenantPgPool.getDefaultConnectOptions().getDatabase(), is("test"));
+  }
+
+  @ParameterizedTest
+  @MethodSource
```

Comment:
`@MethodSource("postgresImage")` might make this more readable since you have to know otherwise that a static method with the same name will be used.